setup.cfg

```diff
@@ -11,7 +11,7 @@ skip = env
 include_trailing_comma = true
 combine_as_imports = True
 force_grid_wrap = 0
-known_third_party = boto3,botocore,cfn_tools,hypothesis,jinja2,jsonschema,nested_lookup,ordered_set,pytest,requests,setuptools,yaml,dpcontracts,aws_lambda_powertools,moto
```

Comment:
What is ``known_third_party`` used for? I saw boto3, botocore,.... are removing now but I dont see code changes related to them.